sktime/annotation/adapters/_pyod.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+from sktime.annotation.base._base import BaseSeriesAnnotator
+
+__author__ = ["mloning", "satya-pattnaik", "fkiraly"]
+
+import pandas as pd
+
+
+class SeriesAnnotatorPyOD(BaseSeriesAnnotator):
```

Comment:
Let's follow the naming conventions that we use for transformers already (estimator scitype at the end, ignoring subtypes)
```suggestion
class PyODAnnotator(BaseSeriesAnnotator):
```

Reply:
sure - generally I'd prefer the scitypes at the start, since then the lexicographic ordering clusters scitypes, but obviously this is the wrong place to change that. I'll comply with current standard.